### PR TITLE
New template design

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,38 +2,38 @@ import angular from 'angular';
...
 	.service(def.SERVICE_NAME, () => () => new Model())
 	.run(Setup)
 	.name;
 
 Setup.$inject = ['$templateCache'];
 function Setup($templateCache) {
-	$templateCache.put('qgrid.html', require('./components/grid/grid.html'));
-	$templateCache.put('qgrid.head.cell.html', require('./components/head/cell.html'));
-	$templateCache.put('qgrid.body.cell.html', require('./components/body/cell.html'));
+	$templateCache.put('qgrid.tpl.html', require('./components/grid/grid.html'));
```
Comment (titovmx) [inline src/index.js:36]: tpl is a template? I don't like this short form.

Comment (klumba12) [reply src/index.js:36]: it's like common extension ('tpl.html'),
please suggest your naming:)

Comment (titovmx) [reply src/index.js:36]: why 'tpl' part is neccessary?
I would prefer 'template.html'

Comment (klumba12) [reply src/index.js:36]: still tpl is the common approach, think about it like about an extension

**Review by titovmx — APPROVED** [review]

**Pull request merged**